packages/plugin-vue-jsx/index.js

```diff
@@ -82,6 +82,7 @@ function vueJsxPlugin(options = {}) {
         }
 
         const result = babel.transformSync(code, {
+          babelrc: false,
```

Comment:
Does this actually need to be specified explicitly for `plugin-vue-jsx`? [Babel docs](https://babeljs.io/docs/en/options#babelrc) suggests that `babelrc` only defaults to true if we provide a `filename` like in `plugin-react-refresh`, and [false otherwise](https://babeljs.io/docs/en/options#filename).

Reply:
I had the same opinion
My idea: we should add a comment referencing a newly created issue to track it for later
_Or just a `TODO` or `FIXME` or so_

Reply:
I think `babelrc`'s default value being dependent on another option is a footgun, so it's good to be explicit here - just in case we need `filename` to be passed here in the future.

Reply:
@yyx990803 hi， @vitejs/plugin-react-refresh haven't upgraded yet？